packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -118,7 +118,7 @@ const StyledSideNav = styled("nav", {
     "&::after": {
       backgroundColor: odysseyDesignTokens.HueNeutral200,
       content: "''",
-      height: "100%",
+      height: "100vh",
```

Comment:
I'm not sure this is best. Sometimes SideNav is presented as less than the full height of the viewport (i.e. if banners at the top of the UI Shell are present).

Reply:
You are correct. This should be `100%`. I think this was a hold over from me experimenting with the scrollbar behavior. Good catch